src/httpserver.cpp

```diff
@@ -478,6 +490,15 @@ void StopHTTPServer()
     LogPrint(BCLog::HTTP, "Stopped HTTP server\n");
 }
 
+bool GetHTTPBindAddress(std::string& address)
+{
+    if (bindAddress.empty()) {
+        return false;
```

Comment:
```cpp
address = bindAddress;
return !address.empty();
```

Reply:
Yes, but that would overwrite the passed in address.  While it doesn't matter in the current context, I found it cleaner to not affect it at all if `false` is returned.